expyfun/stimuli/_tracker.py

```diff
@@ -491,7 +516,7 @@ class TrackerBinom(object):
     stop_early : boolean
         Whether to stop the adaptive track as soon as ``alpha`` is reached and
         at least ``min_trials`` have been presented.
-    x_current : float | None
+    x_current : float | np.nan
         The level that you want to run the test at. This has no bearing on how
         the track runs, and it will never change, but it is required to be
         here for ``TrackerDealer``.
```

Comment:
`nan` is already a `float`, no need to distinguish

I'm a bit confused by this. Why does it need to be a parameter if it has no bearing on the values / outcomes? Is this from an earlier version back when `TrackerDealer` would do the initialization of its trackers (which is no longer supported I think)?